spec/Recruiter/Job/RepositoryTest.php

```diff
@@ -132,6 +130,258 @@ public function testCountsQueuedJobsGroupingByASpecificKeyword()
         );
     }
 
+    public function testGetDelayedScheduledJobs()
```

Comment:
@silvadanilo @gabrielelana is this naming okay for you?

Reply:
ok